sdk/lib/convert/ascii.dart

```diff
@@ -195,6 +204,31 @@ abstract class _UnicodeSubsetDecoder extends Converter<List<int>, String> {
   Stream<String> bind(Stream<List<int>> stream) => super.bind(stream);
 }
 
+/// This class converts ASCII bytes to strings
```

Comment:
I don't think we need "This class"?

It can just go straight in to the description:  "Converts ASCII bytes to strings."

I think that's the style that other Dart docs use?